### ISSUE TITLE
Input

### ISSUE THREAD

**Issue by daveaneo** [bitcoinroyale/whitepaper#9]
You are a good writer!

1. "gold - susceptibility to theft"  — use em-dash not hyphen.
2. "Litecoin [2] that," "Bitcoin Cash [3] that," and "Bitcoin Gold [4] that" —  use "which," instead of "that."
3. "10 minute confirmation times" — 10-minute confirmation times.
4. "long standing industry" — long-standing industry.
5. "on a single goal - creating the ultimate store of value."  — use em-dash not hyphen.
6. "Since we would not expect to freight a shipment of gold across locations in under 10 minutes anyways, this sacrifice seems natural." This sentence seems off to me.
7. "well-versed in the latest security" — no hyphen.
8. "off a less attractive form of cash with a very attractive form of gold that cannot be stolen" — it's not invulnerable.

Questions:

A. If 100 bitcoins were sent (Transaction Alert) by a hacker, the wallet owner would be able to invalidate this with a single transaction (Recovery Transaction). But what if the hacker sent a million transactions? Would the wallet owner be able to defend against them all within the 24 hours -- or worth paying for them?

B. Is it possible for the Recovery Transaction to invalidate all Transactions Alerts from a single wallet by using a single transaction? 

C. 24 hours may go by without checking one's wallet. What about a passive defense? What if there was a parameter which required a special transaction within 24 - 48 hours any time the total 24-hour outgoing Transaction Alert exceeded some amount? Every 24 hours, the outgoing amount is calculated.

BTC: 1ASyovLk4EKA6hL5LvDNNcTaJ5DyEojbka

**Comment by ianduoteli** [comment]
Regarding question A:

That's a good edge case.. Let's assume the hacker creates 1 million transactions. All of these transactions must depend on existing UTXOs of the wallet owner as inputs. Let's assume the wallet owner had 10 separate UTXOs that hold all of their funds before the attack. The hacker will take these 10 UTXOs and create new UTXOs, and from them more and more, until they create all 1 million transactions.

It's enough for the wallet owner to create recovery transactions just for the 10 original UTXOs. This small number of recovery transactions will spend these 10 UTXOs (by sending them to a new wallet for example). This will invalidate all existing transaction alerts that depend on these UTXOs as input. And of course will invalidate any transactions depending on outputs of the invalidated transactions. 

Bottom line, all 1 million hacker transactions will be invalidated just by making a small number of recovery transactions (spending only the original UTXOs the hacker relied on).
